src/events/requests/NewRequestEventHandler.ts

```diff
@@ -28,6 +29,53 @@ export default class NewRequestEventHandler implements EventHandler {
 
 		this.logger.info( `User ${ origin.author.tag } posted a new request to requests channel ${ origin.channel.id }` );
 
+		this.handleNewRequest( origin );
+	};
+
+	// This syntax is used to ensure that `this` refers to the `NewRequestEventHandler` object
+	public onReopen = async ( reaction: MessageReaction, user: User ): Promise<void> => {
+		this.logger.info( `User ${user.tag} is reopening the request message '${reaction.message.id}'` );
```

Comment:
That's some spaces for `${user.tag}` and the other one, `lint` won't mark you, but they are generally there in other parts of the code.
I don't think they are necessary but I am forcing myself to add those now as well, to keep it consistent
Not gonna mark them in other instances tho.

Reply:
Yeah, generally I think it's good to have spaces in these places, but it ESLint doesn't have the rule about this enabled yet. I think I'll configure the rule and change the existing code when merging so that it'll have spaces everywhere, just like with the brackets.